prototype/2_register/index.js

```diff
@@ -0,0 +1,6 @@
+// save name and email to local storage on form submit
+document.getElementById('submit').addEventListener('click', function(e) {
```

Comment:
You never used the function argument in the code, therefore it does not need to be there